docs/components/breadcrumbs.md

```diff
@@ -231,6 +289,11 @@ export default defineComponent({
     font-size: 13px;
     font-weight: 300;
     line-height: 14px;
-    color: var(--kui-color-text-neutral-weak, $kui-color-text-neutral-weak);
+    color: #afb7c5;
```

Comment:
question: why did you remove a token?

Reply:
@portikM 's guide said not to use tokens in the docs

Reply:
* correction, we're definitely not using `var()` in docs. If you want - you can use the SCSS variable like so:
```
color: $kui-color-text-neutral-weak;
```

But the sure way is to just use hardcoded value - perfectly fine in docs.